### ISSUE TITLE
over-fit issue

### ISSUE THREAD

**Issue by liyuan-qiu** [zju3dv/NeuralRecon#148]
I have been training in my own datasets, but now the overfitting issue can't be fixed by adding new balanced datasets. The overfitting issue is two-directional, in some training, the 3D model will be over-complete, but some training, the model is under-complete. Which parameter can be adjust to set some lower or upper bound to fix the overfitting issue? Or do I need to adjust the loss function?
